.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,216 @@
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('project/backend/**/build.gradle', 'project/backend/**/gradle-wrapper.properties') }}
+          restore-keys: ${{ runner.os }}-gradle
+      
+      - name: Update JaCoCo configuration in build.gradle files
+        run: |
+          for dir in project/backend/*/; do
+            if [ -f "$dir/build.gradle" ]; then
+              echo "Updating JaCoCo config in $dir/build.gradle"
+              # Backup the original file
+              cp "$dir/build.gradle" "$dir/build.gradle.bak"
+              
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)